quill-core/src/main/scala/io/getquill/sources/BindedStatementBuilder.scala

```diff
@@ -24,13 +24,22 @@ class BindedStatementBuilder[S] {
     this
   }
 
-  def build(query: String) = {
+  def build(query: String): (String, (S => S)) = {
 
     @tailrec
     def expand(q: List[Char], b: List[Binding[S]], acc: List[Char]): List[Char] =
       (q, b) match {
         case (Nil, Nil) =>
           acc
+        case ('\'' :: qtail, b) => // skip quote
```

Comment:
Postgres accept `"Test?"` as table name, and mysql allow `?` inside backit as table name. Though I guess no one would name table/column like that. 

And situation goes more complicated if we consider `select * from TestEntity where s = '\'?'` which was valid in mysql